### PR TITLE
otto 0.1.2 (new formula)

### PR DESCRIPTION
**Pull request by markcornick**
This builds on the work in #44469, updating for the most recent version of otto and completing the list of go_resources.

### CODE REVIEW

File: Library/Formula/otto.rb [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    end
+
+    cd gopath/"src/github.com/hashicorp/otto" do
+      system "make", "dev"
+      bin.install "bin/otto"
+    end
+  end
+
+  test do
+    system "#{bin}/otto", "--version"
```
Comment (mikemcquaid) [inline Library/Formula/otto.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by markcornick** [comment]
Respectfully, no. Please see https://github.com/Homebrew/homebrew/pull/44469#issuecomment-145212198 - testing, say, "otto dev" would require setting up Vagrant, Virtualbox, and a Rails or similar app (with all of its dependencies) at which point we would need to test a lot of things that ultimately have nothing to do with this formula - essentially it becomes a Vagrant/Virtualbox integration test. There isn't really a good command, at this point in otto's development, to substitute in this test. Perhaps this can be revisited in the future, but for now, will this do? Thanks.

**Comment by mikemcquaid** [comment]
> Respectfully, no. Please see #44469 (comment) - testing, say, "otto dev" would require setting up Vagrant, Virtualbox, and a Rails or similar app (with all of its dependencies) at which point we would need to test a lot of things that ultimately have nothing to do with this formula - essentially it becomes a Vagrant/Virtualbox integration test. There isn't really a good command, at this point in otto's development, to substitute in this test. Perhaps this can be revisited in the future, but for now, will this do? Thanks.
>  Show all checks

That's OK, thanks for explaining!

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by joshRpowell** [comment]
thanks @markcornick for this contribution!  
fyi...a new version of otto was pushed yesterday. https://github.com/hashicorp/otto/releases/tag/v0.2.0

**Comment by bfontaine** [comment]
@joshRpowell Feel free to make a pull-request :wink: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md

**Comment by markcornick** [comment]
Please don't necro-post on pull requests. Thanks.

**Pull request closed without merging**